src/main/java/io/appform/dropwizard/actors/router/config/HierarchicalOperationWorkerConfig.java

```diff
@@ -0,0 +1,39 @@
+package io.appform.dropwizard.actors.router.config;
+
+import io.appform.dropwizard.actors.actor.ConsumerConfig;
+import io.appform.dropwizard.actors.actor.ProducerConfig;
+import lombok.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+@Data
+@EqualsAndHashCode
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class HierarchicalOperationWorkerConfig {
+
+    @Min(1)
+    @Max(100)
+    @Builder.Default
+    private int concurrency = 3;
+
+    @Min(1)
+    @Max(100)
+    @Builder.Default
+    private int prefetchCount = 1;
+
+    @Min(2)
+    @Max(32)
+    private Integer shardCount;
+
+    @Valid
+    private ProducerConfig producer;
+
+    @Valid
+    private ConsumerConfig consumer;
+
```

Comment:
@AshayKing Can you please expose all other relevant fields as well like Retry strategy in the config.